pkg/networkmanager/wireguard.jsx

```diff
@@ -0,0 +1,192 @@
+                            </Button>
+                        }
+                    />
+                }
+                className='dynamic-form-group'
+            >
+                {(peers.length !== 0)
+                    ? peers.map((peer, i) => (
+                        <Grid key={i} hasGutter>
+                            <FormGroup className='pf-m-6-col-on-md' label={_("Public key")} fieldId={idPrefix + '-publickey-peer-' + i}>
+                                <TextInput
+                                    value={peer.publicKey}
+                                    onChange={(_, val) => {
+                                        setPeers(peers => peers.map((peer, index) => i === index ? { ...peer, publicKey: val } : peer));
+                                    }}
+                                    id={idPrefix + '-publickey-peer-' + i}
+                                />
+                            </FormGroup>
+                            <FormGroup className='pf-m-3-col-on-md' label={_("Endpoint")} fieldId={idPrefix + '-endpoint-peer-' + i}>
+                                <TextInput
+                                    value={peer.endpoint}
+                                    onChange={(_, val) => {
+                                        setPeers(peers => peers.map((peer, index) => i === index ? { ...peer, endpoint: val } : peer));
+                                    }}
+                                    id={idPrefix + '-endpoint-peer-' + i}
+                                />
+                            </FormGroup>
+                            <FormGroup className='pf-m-3-col-on-md' label={_("Allowed IPs")} fieldId={idPrefix + '-allowedips-peer-' + i}>
+                                <TextInput
+                                    value={peer.allowedIps}
+                                    onChange={(_, val) => {
+                                        setPeers(peers => peers.map((peer, index) => i === index ? { ...peer, allowedIps: val } : peer));
+                                    }}
+                                    id={idPrefix + '-allowedips-peer-' + i}
+                                />
+                            </FormGroup>
+                            <FormGroup className='pf-m-1-col-on-md remove-button-group'>
+                                <Button
+                                    variant='secondary'
+                                    onClick={() => {
+                                        setPeers(peers => peers.filter((_, index) => i !== index));
+                                    }}
+                                >
+                                    <MinusIcon />
+                                </Button>
+                            </FormGroup>
+                        </Grid>
+                    ))
+                    : <EmptyState>
+                        <EmptyStateBody>{_("No peers added.")}</EmptyStateBody>
+                    </EmptyState>
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230627-160728-be631904-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)